packages/web-components/src/components/va-maintenance-banner/va-maintenance-banner.tsx

```diff
@@ -0,0 +1,150 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+import classNames from 'classnames';
+import {
+  formatDate,
+  isDateAfter,
+  isDateBefore,
+  isDateSameDay,
+} from '../../utils/date-utils';
+
+/**
+ * @componentName Maintenance Banner
+ * @maturityCategory use
+ * @maturityLevel deployed
+ * @guidanceHref form/maintenance-banner
```

Comment:
Should this have a form href reference? I think this is for linking it on the design.va.gov documentation site under the [forms dropdown](https://design.va.gov/components/form/) but I'm not sure if this component will be there.

Reply:
removed guidanceHref